src/api/viya/getFileStream.ts

```diff
@@ -0,0 +1,16 @@
+import { isFolder } from '@sasjs/utils/file'
+import { generateTimestamp } from '@sasjs/utils/time'
+import { Job } from '../../types'
+
+export const getFileStream = async (job: Job, filePath?: string) => {
```

Comment:
Moving this functionality into a separate file lets us remove the `fs` import in `pollJobState.ts`.
The `require` on line 6 also makes sure we only import the `createWriteStream` function when it is actually needed.